staff/mgarciarojas/Javascript exercises/Calculation.js

```diff
@@ -0,0 +1,16 @@
+//Calculation
+//Write a program that writes in the console all the multiples of 23 less than 500 and at the end writes the sum of all of them
+
+function calculation(){
+    var acc=0;
+    var table=""
+    for (i=0; i<500; i++){
+        if (i%23 == 0){
+            table+=i+ "\t"
+           acc+=i
+        } 
+    }
+    return table
+    return acc       
```

Comment:
Do you think a function can `return` two times? Try to assign the result of this function to a variable and check its value